patch for code-signing (#847)
codemagic-ci-cd/codemagic-docs: content/yaml-code-signing/code-signing-identities.md

Codemagic provides two means of fetching the required certificates and provisioning profiles during the build with the use of `codemagic.yaml`. Fetching can either be configured by specifying the distribution type and bundle identifier, or for more advanced use-cases, individual files can be fetched by their reference names.

#### Options 1: Fetching files by distribution type and bundle identifier

To fetch all uploaded signing files matching a specific distribution type and bundle identifier during the build, define the `distribution_type` and `bundle_identifier` fields in your `codemagic.yaml` configuration. Note that it is necessary to configure both of the fields.

```yaml
environment:
  ios_signing:
    distribution_type: ad_hoc # app_store | development | enterprise
    bundle_identifier: com.example.id
scripts:
  ......

>   - name: Set up keychain to be used for codesigning using Codemagic CLI 'keychain' command

Reviewer: ```suggestion
  - name: Initialize keychain
```

    script: |
      keychain initialize

>   - name: Use system default keychain

Reviewer: ```suggestion
  - name: Add certificates to keychain
```

    script: |
      keychain add-certificates
  ......
```

Note that when using the fields `distribution_type` and `bundle_identifier`, it is not allowed to configure `provisioning_profiles` and `certificates` fields.

> Additionally it is mandatory to specify `Setup Keychain` & `Add keychain` scripts as show above.

Reviewer: ```suggestion
Additionally, it is mandatory to specify `Initialize keychain` & `Add certificates to keychain` scripts as shown above.
```
